ZupahBank/BusinessLib/Interfaces/ICustomerManagement.cs

```diff
@@ -0,0 +1,18 @@
+﻿using BusinessLib.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLib.Interfaces
+{
+    interface ICustomerManagement
+    {
+        void Create(string name, string legalId);
+        void Delete();
```

Comment:
Which object will be deleted here? Maybe the method should have some input parameters.